CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+/src @BGDanny
```

Comment:
JW, do we need the slash at the end of the path here. Because according to the doc
![image](https://user-images.githubusercontent.com/60146634/190484216-27ac1b2f-9940-4dea-bd71-c48f454b12af.png)

Reply:
Good catch, I fixed it!